mac

```diff
@@ -46,6 +46,16 @@ fi
 # shellcheck disable=SC2016
 append_to_zshrc 'export PATH="$HOME/.bin:$PATH"'
 
+if [ -d "/usr/local" ]; then
+  if ! sudo -u "$(whoami)" [ -r /usr/local ]; then
```

Comment:
This line looks like it's using `sudo` to run `[ -r /usr/local ]` as...the current user.

How is this different from the command without sudo, i.e. `if ! [ -r /usr/local ]`